swagger.yaml

```diff
@@ -412,4 +438,34 @@ definitions:
           nextPage:
             type: string
 
-
+  ApplicationResponse:
+    type: 'object'
+    properties:
+      application:
+        type: 'object'
+        properties:
+          clientGeneratedId:
+            type: 'string'
+          applicationDate:
+            type: 'string'
+            format: 'date-time'
+          status:
+            type: 'string'
```

Comment:
With the exception of the additional `applicationDate` and `status` properties, this would be the API returning the data in the same form as it was submitted.

Is this what we want? Or do we need to create new definitions?

Reply:
Yes - we should be as symmetrical as possible - with additions.
AFAIK: Additions will also be validations, and notes.

Reply:
I realised that if we use the same definitions as the ones used for submitting an application, they will include criteria like `minLength: 6`. I'm not certain if that's correct to include in the definition for what we are returning from the API? Do we mind about that? I don't really want to duplicate all those definitions, and maybe it's still useful information in the context of an API response?

Reply:
No we don't mind at this point. Theoretically speaking, they should hold true